### PR TITLE
{bio} [goolf-1.4.10] SPAdes-3.6.2 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/f502f36ed36f70697be3 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/a2da711b9d70fa369a27 for a full test report.

File: easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = "CMakeMake"
+
+name = 'SPAdes'
+version = '3.6.2'
+
+homepage = 'http://bioinf.spbau.ru/en/spades'
+description = """ Genome assembler for single-cell and isolates data sets """
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb]: drop the whitespace before and after?

File: easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+sanity_check_commands = [('spades.py', '--test')]
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["bwa-spades", "dipspades", "dipspades.py",
+                                     "hammer", "ionhammer", "spades",  "spades.py"]],
+    'dirs': [],
+}
+
+moduleclass = 'bio'
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb]: drop empty line

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7ddb97471511ec87757c for a full test report.

**Comment by boegel** [comment]
@pescobar: doesn't work for me, is Boost missing as a dep?

**Comment by boegel** [comment]
@pescobar ping on missing Boost dep?

File: easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+description = """Genome assembler for single-cell and isolates data sets"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://spades.bioinf.spbau.ru/release%(version)s/']
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [('CMake', '2.8.12')]
+
+dependencies = [('zlib', '1.2.8')]
```
Comment (boegel) [inline easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb]: is bzip a missing dep here too?

cfr. https://lists.ugent.be/wws/arc/easybuild/2016-04/msg00055.html

**Comment by pescobar** [comment]
@boegel weird your issue with boost because the changelog says "no dependencies on boost"
http://spades.bioinf.spbau.ru/changelog.html

**Comment by pescobar** [comment]
@boegel can you run a test on this one? https://github.com/hpcugent/easybuild-easyconfigs/pull/3135

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/43edebea66cfe57fe227301094de66fb for a full test report.

**Comment by boegel** [comment]
@pescobar maybe you need to specify `-DSPADES_BOOST_ROOT` (to `EBROOTBOOST/include`?):

```
$ tail -9 SPAdes-3.6.2/src/cmake/deps.cmake

# Use included boost unless explicitly specified
if (NOT SPADES_BOOST_ROOT)
  set(BOOST_ROOT "${EXT_DIR}/include")
else()
  set(BOOST_ROOT SPADES_BOOST_ROOT)
endif()
set(Boost_USE_MULTITHREADED ON)
find_package(Boost REQUIRED)
```

**Comment by pescobar** [comment]
@boegel thanks for the tip. Would you retest?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5e1342edf97e5bc58fca8951b89e1bf8 for a full test report.

File: easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    ('zlib', '1.2.8'),
+    ('bzip2', '1.0.6'),
+    ('Boost', '1.53.0'),
+]
+
+start_dir = 'src'
+
+separate_build_dir = True
+
+configopts = '-DSPADES_BOOST_ROOT=${EBROOTBOOST}'
```
Comment (boegel) [inline easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb]: @pescobar I think you need to use `/include`...

**Comment by pescobar** [comment]
@boegel I tried a workaround adding `-DBoost_NO_BOOST_CMAKE=ON`

would you retest....again :)

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/2d6c5e3e1a9d93ad392912b0fd683b84 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6fa7c6a5aa1bb95ddc23e4cfa7caf2bd for a full test report.

**Comment by pescobar** [comment]
I think this one is ready to merge

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**